dev-libs/libtar is automatically detected, bug #154194
this patch allows to disable libtar support, was is done by default
Patch status : submitted upstream, https://trac.videolan.org/vlc/ticket/1045
Commited in vlc trunk by xtophe Feb. 11 2007
===================================================================
--- vlc-0.8.6_p18636.orig/configure.ac
+++ vlc-0.8.6_p18636/configure.ac
@@ -2923,10 +2923,15 @@ AC_CHECK_HEADERS(sysfs/libsysfs.h, [
 dnl
 dnl skins2 module
 dnl
-AC_CHECK_HEADERS(libtar.h, [
-  VLC_ADD_LDFLAGS([skins2],[-ltar])
-] )
+AC_ARG_ENABLE(libtar,
+  [  --enable libtar support for skins2 (default: check)])
 
+if test "${enable_libtar}" != "no"
+then
+  AC_CHECK_HEADERS(libtar.h, [
+    VLC_ADD_LDFLAGS([skins2],[-ltar])
+  ] )
+fi
 
 
 dnl
